### PR TITLE
cloud: remove v1.1 pages, update links

### PR DESCRIPTION
**Pull request by danieldides**
These pages existed for historical reasons and SOC2 certification. Now that Cloud v2 is fully certified and there are no longer any running v1.1 instances, we do not need these pages anymore. They cause false search results and are generally misleading.

Security has [approved](https://sourcegraph.slack.com/archives/C1JH2BEHZ/p1689952937030599) removing these pages:
Screenshot 2023-08-01 at 11 46 42

### CODE REVIEW

File: content/departments/cloud/technical-docs/index.md [hunk 1]
```diff
@@ -15,25 +15,14 @@ Sourcegraph upgrades every test and customer instances according to [SLA](#slas-
...
 4. Cloud team performs upgrade of all instances in given order:
 
-- for Instances with version [v1.1](./v1.1/mi1-1_upgrade_process.md)
-  Stage | Working days since release | Action | Condition not met?
-  --- | --- | --- | --- |
-  1 | 0-2 | Upgrade internal instances by Cloud Team (incl. [demo](https://demo.sourcegraph.com/) and [rctest](https://rctest.sourcegraph.com/)) |
-  2 | 3-4 | Time for verification by Sourcegraph teams | New patch created -> start from 1st stage
-  3 | 5-6 | Upgrade: 30% trials 10% customers | New patch created -> upgrade internal in 1 working day and start from 2nd stage
-  4 | 7-8 | Upgrade: 100% trials 40% customers | New patch created -> upgrade internal in 1 working day and start from 3rd stage
-  5 | 9-10 | Upgrade: 100% customers | New patch created -> upgrade internal in 1 working day and start from 3rd stage
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/index.md:29]: this release schedule is still true, we should keep it

Comment (danieldides) [reply content/departments/cloud/technical-docs/index.md:29]: Already added it back in 😉

File: content/departments/cloud/technical-docs/index.md [hunk 2]
```diff
@@ -28,12 +27,10 @@ The release process is performed in steps:
   4 | 7-8 | Upgrade: 100% trials 40% customers | New patch created -> upgrade internal in 1 working day and start from 3rd stage
   5 | 9-10 | Upgrade: 100% customers | New patch created -> upgrade internal in 1 working day and start from 3rd stage
 
-After upgrade of every single instances [Uptime checks](./v1.1/mi1-1_upgrade_process.md#confirm-instance-health) are verified. This includes [automated monitoring](#monitoring-and-alerting)
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/index.md:31]: we should keep this, but maybe just remove the link to what is uptime checks

it's part of the control we do post-upgrade monitoring

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ You can learn more about the detail from the following RFCs:
...
 
-Every instance has a default Sourcegraph admin user added during the [instance initialisation](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/7e9066e537b02feb6013585d443fc27514b71a71/util/cmd/mg_init_instance.go#L51-L54). The [username, password and access token](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/36db9bb65ec72ffa470752461b82c6999c00b969/util/pkg/config/config.go#L47-L49) of the admin user is stored in Google Secret Manager (GSM) in the GCP project of the managed instance. The access token is used to access the managed instance by our services programmatically.
+Every instance has a default Sourcegraph admin user added during the [instance initialisation](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/controller/-/blob/internal/instances/init.go?L33-57). The [username, password and access token](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/controller/-/blob/internal/instances/init.go?L124-138) of the admin user is stored in Google Secret Manager (GSM) in the GCP project of the managed instance. The access token is used to access the managed instance by our services programmatically.
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]: nit: use permalink

**Review by michaellzc — APPROVED** [review]

**Pull request merged**